imagebuildah/build.go

```diff
@@ -483,6 +483,19 @@ func (s *StageExecutor) volumeCacheRestore() error {
 // imagebuilder tells us the instruction was "ADD" and not "COPY".
 func (s *StageExecutor) Copy(excludes []string, copies ...imagebuilder.Copy) error {
 	for _, copy := range copies {
+		// If the file exists, check to see if it's a symlink.
+		// If it is a symlink, convert to it's target otherwise
+		// the symlink will be overwritten.
+		fileDest, _ := os.Stat(s.mountPoint + copy.Dest)
```

Comment:
You probably want `Lstat()`, not Stat(). But even if I change that on my end, I get FileMode `1000000777` which makes no sense. Deferring this again for a while, sending back your way. HTH.

Reply:
yep, changed to Lstat.  Made the other changes too, thanks @edsantiago